server/config/config_test.go

```diff
@@ -47,7 +47,12 @@ func TestConfigRoundtrip(t *testing.T) {
 			case int:
 				key_v.SetInt(int64(conf_index*100 + key_index))
 			case bool:
-				key_v.SetBool(true)
+				switch conf_v.Type().Field(key_index).Name {
+				case "UseTLS":
```

Comment:
I believe that this test failure actually caught an issue with how the configuration is handled. I am digging into this.